### DISCUSSION TITLE
Default to vanilla gnome plus core packages

### DISCUSSION THREAD

**Discussion by carnage-mode** [AerynOS/discussions#6] [Ideas]
I think it's best to leave all sorts of customization to the users, providing a clean slate we can build upon. This is what makes Fedora very appealing to someone like me. I know this probably won't be a priority right now but it could be something to look into in the future.
These are the gnome recommended packages:
https://gitlab.gnome.org/GNOME/gnome-build-meta/-/tree/master/elements/core?ref_type=heads

**Comment by ikeycode** [comment]
Yeah tbh we're not looking to heavily patch anything it only makes it harder to update the system. Build and integration sure
